libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/bulletChart/tests/PluggableBulletChart.test.tsx

```diff
@@ -313,6 +313,10 @@ describe("PluggableBulletChart", () => {
                             referencePointMocks.masterMeasureItems[0],
                         ],
                     },
+                    {
```

Comment:
Well, I originally thought that in case that you do not date bucket items you are not able to create PoP and therefore I edited these mocks (I thought that it was just a bad mock and it worked because there was not any check for date datasets).  Then I realized that you are able to create PoP when you have just metrics in insight 🤦 so you had a really good point. Thanks.

Fix here: https://github.com/gooddata/gooddata-ui-sdk/pull/982/commits/31f7f92be03e07b4d8fbdc70aa88cd0fd44f5ec1